ui/pages/swaps/view-quote/view-quote.js

```diff
@@ -1022,10 +1024,12 @@ export default function ViewQuote() {
                 fee: feeInEth,
                 maxFee: maxFeeInEth,
               }}
-              secondaryFee={{
-                fee: feeInFiat,
-                maxFee: maxFeeInFiat,
-              }}
+              secondaryFee={
+                useCurrencyRateCheck && {
+                  fee: feeInFiat,
+                  maxFee: maxFeeInFiat,
```

Comment:
should we keep the maxFee here and show it in the current currency? 

w/ currency rate setting turned off:
<img width="520" alt="Screen Shot 2023-01-16 at 11 18 17 PM" src="https://user-images.githubusercontent.com/20778143/212731918-fad77c30-a530-46ed-91a2-5260daf24b22.png">

w/ currency rate setting turned on: 
<img width="519" alt="Screen Shot 2023-01-16 at 11 17 11 PM" src="https://user-images.githubusercontent.com/20778143/212732109-e5c2a538-913f-4c53-8fb9-71069a265640.png">

Reply:
When the currency rate checking is disabled, we will stop fetching the conversion rates, so the max fee displayed above will not be accurate.

Reply:
yeah, best not to show fiat values when the currency rating check is disabled

Reply:
still wrapping my head around the logic. I think I understand that turning off the currency rate setting will stop the polling process. I thought this was related to fetching currency rates to check against the network currency, and that this is separate from fetching the current network currency data. `getRenderableNetworkFeesForQuote` in the screenshot below appears to provide the maxFee in Fiat and the currency network currency. Is the `maxFeeInEth` impacted by the new setting? 

<img width="1472" alt="Screen Shot 2023-01-17 at 1 14 05 AM" src="https://user-images.githubusercontent.com/20778143/212746383-d9c82b77-ff0c-4e99-b195-f9439b184e5a.png">
<img width="961" alt="Screen Shot 2023-01-17 at 1 20 50 AM" src="https://user-images.githubusercontent.com/20778143/212746410-c5878bc2-97e3-4bac-979d-62f9bbb8f908.png">

Reply:
@NiranjanaBinoy explained to me that turning off the setting
> blocks the polling for conversion rates from CurrencyRateController and TokensRateController. So if the toggle is OFF the update values for conversion rates will not be pulled in and any value that is still in state will be out dated.

I think I see how the values shown can be outdated now.

Thanks for the explanation @NiranjanaBinoy! marking this as resolved

Reply:
@NiranjanaBinoy explained to me that turning off the setting

blocks the polling for conversion rates from CurrencyRateController and TokensRateController. So if the toggle is OFF the update values for conversion rates will not be pulled in and any value that is still in state will be outdated.

I think I see how the values shown can be outdated now. Thanks for this explanation @NiranjanaBinoy! 

I'm going to take a bit more time to understand this more.

Reply:
@digiwand I have updated the code to use the maxFeeInEth for max Fee in the swaps page. After discussing with @danjm, it was decided to add the maxFeeInEth when the toggle is OFF. Thanks for raising the concern ❤️

Reply:
awesome! Thanks for looking into it more w/ Dan. LGTM ❤️